defaults/test_default.yaml

```diff
@@ -195,3 +195,5 @@ scylla_apt_keys:
 raid_level: 0
 
 bare_loaders: false
+
+exclude_disabled_nemesis: true
```

Comment:
please rename to `nemesis_exclude_disable`
i.e. it's would be easier to find when it's prefix with `nemesis`

Reply:
fixed in next push.

Reply:
In theory I would suggest doing  `nemesis_selector: [!disable]` as the default, problem is that our configuration is always overwriting and not merging lists, so all the places currently using `nemesis_selector` would need to be changed and tracked. so adding configuration is acceptable.

adding more nemesis classes for selection of test, is also cumbersome, and need changes in multiple places (and doesn't leave you control of configured it differently)